locales/en.default.schema.json

```diff
@@ -555,7 +559,7 @@
         },
         "header__1": {
           "content": "Email Signup",
-          "info": "Subscribers added to your “accepted marketing” [customer list.](https://help.shopify.com/en/manual/customers/manage-customers)"
+          "info": "Subscribers added automatically to your “accepted marketing” [customer list.](https://help.shopify.com/en/manual/customers/manage-customers)"
```

Comment:
Thank you for having "eagle eyes"! 

The dot had to stay inside the link after all to not make a break with the icon. Melissa caught it this morning. (Sorry!)

Agree since this is going to help (vs into customer account area / the list itself):
`"info": "Subscribers added automatically to your “accepted marketing” customer list. [Learn more] (https://help.shopify.com/en/manual/customers/manage-customers)"`